pwa/app/lib/utils.ts

```diff
@@ -8,3 +9,58 @@ export function cn(...inputs: ClassValue[]) {
 export function parseDateString(date: string) {
   return new Date(date);
 }
+
+export function sortMatchComparator(a: Match, b: Match) {
+  const compLevelValues: Record<string, number> = {
+    f: 5,
+    sf: 4,
+    qf: 3,
+    ef: 2,
+    qm: 1,
+  };
+  if (a.comp_level !== b.comp_level) {
+    return (
+      (compLevelValues[a.comp_level] ?? 0) -
+      (compLevelValues[b.comp_level] ?? 0)
+    );
+  }
+
+  return a.set_number - b.set_number || a.match_number - b.match_number;
+}
+
+export function sortMatches(matches: Match[]) {
```

Comment:
Is this used?

Reply:
Not currently, it was originally but then I just wound up using the comparator function instead. I'll remove